server/src/server.common.ts

```diff
@@ -0,0 +1,166 @@
+
+import {
+	createConnection,
+	TextDocuments,
+	Diagnostic,
+	DiagnosticSeverity,
+	ProposedFeatures,
+	InitializeParams,
+	DidChangeConfigurationNotification,
+	CompletionItem,
+	CompletionItemKind,
+	TextDocumentPositionParams,
+	TextDocumentSyncKind,
+	InitializeResult,
+	_Connection,
+} from 'vscode-languageserver';
+
+import {
+	TextDocument
+} from 'vscode-languageserver-textdocument';
+
+import { validator, errors } from '@openfga/syntax-transformer';
+
+export function startServer(connection: _Connection) {
+	// Create a simple text document manager.
+	const documents: TextDocuments<TextDocument> = new TextDocuments(TextDocument);
+
+	let hasConfigurationCapability = false;
+	let hasWorkspaceFolderCapability = false;
+	let hasDiagnosticRelatedInformationCapability = false;
+
+	connection.onInitialize((params: InitializeParams) => {
+		const capabilities = params.capabilities;
+
+		// Does the client support the `workspace/configuration` request?
+		// If not, we fall back using global settings.
+		hasConfigurationCapability = !!(
+			capabilities.workspace && !!capabilities.workspace.configuration
+		);
+		hasWorkspaceFolderCapability = !!(
+			capabilities.workspace && !!capabilities.workspace.workspaceFolders
+		);
+		hasDiagnosticRelatedInformationCapability = !!(
+			capabilities.textDocument &&
+			capabilities.textDocument.publishDiagnostics &&
+			capabilities.textDocument.publishDiagnostics.relatedInformation
```

Comment:
Can you make sure that this is returning true?

Reply:
True for node & web versions when running from launch.